tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -15,37 +14,102 @@ ${pachyderm_appname}            pachyderm
+    Wait Until Status Is Running
 
-Dashboard Test Teardown
+Pachyderm Test Teardown
+    Go To    ${OCP_CONSOLE_URL}
+    Delete Tabname Instance For Installed Operator    ${pachyderm_container_name}   ${pachyderm_container_name}     ${pachyderm_appname}
+    Uninstall Operator    ${pachyderm_operator_name}
+    Delete Project By Name      pachyderm
+    Launch Dashboard    ocp_user_name=${TEST_USER.USERNAME}    ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}
+    ...    browser_options=${BROWSER.OPTIONS}
+    Remove Disabled Application From Enabled Page    app_id=pachyderm
     Close All Browsers
 
 Wait Until Status Is Running
     [Documentation]     Checks if the status changes from Initializing to Running.
     Wait Until Keyword Succeeds     120     1       Element Text Should Be    //span[@data-test="status-text"]      Running
 
+Get Pachd Version
+    Menu.Navigate To Page       Operators       Installed Operators
+    Select Project By Name      ${pachyderm_appname}
+    Click On Searched Operator      ${pachyderm_operator_name}
+    Switch To New Tab       ${pachyderm_operator_name}
+    Wait Until Page Contains Element    //a[@data-test-operand-link="pachyderm-sample"]
+    Click Element   //a[@data-test-operand-link="pachyderm-sample"]
+    Wait Until Page Contains Element    (//dd[@data-test-selector="details-item-value__Version"])[1]
+    ${version}=     Get Text        (//dd[@data-test-selector="details-item-value__Version"])[1]
+    ${seperator}=   Set Variable    v
+    ${res_version}=     Fetch From Right    ${version}      ${seperator}
+    [Return]    ${res_version}
+
 Create Pachyderm AWS-Secret
     [Documentation]     Creates a Pachyderm AWS Secret.
     Run     oc create secret generic pachyderm-aws-secret -n pachyderm --from-literal=access-id=${S3.AWS_ACCESS_KEY_ID} --from-literal=access-secret=${S3.AWS_SECRET_ACCESS_KEY} --from-literal=region=us-east-1 --from-literal=bucket=ods-ci-pachyderm
     Menu.Navigate To Page       Workloads   Secrets
-    Wait Until Page Contains Element        //input[@data-test-id="item-filter"]
+    Wait Until Page Contains Element        //input[@data-test-id="item-filter"]            10
     Input Text      //input[@data-test-id="item-filter"]    pachyderm-aws-secret
     Wait Until Page Contains Element        //a[@data-test-id="pachyderm-aws-secret"]       10
+
+Verify Pipline Pod Creation
+    ${status}=    Check If POD Exists    pachyderm      app=pipeline-edges-v1
+    Run Keyword IF    '${status}'=='FAIL'    FAIL
+    ...    PODS with Label '${label_selector}' is not present in '${namespace}' namespace
+    Wait Until Keyword Succeeds     120     5   Verify Operator Pod Status  pachyderm   app=pipeline-edges-v1
+
+Create Command In Multiple Lines
```

Comment:
This keyword is just used once and could be replaced by a variable with the value. [Here](https://github.com/red-hat-data-services/ods-ci/blob/ff28658ed50564ab48125322dd1b7ac53e2fd41b/tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot#L21) you'll find an example how to do it  (you might have to tweak the  SEPARATOR parameter)

Reply:
I have created a keyword so that it don't creates a mess in another keyword which has nothing to do with it. So I think this should be fine.